### PR TITLE
[W5.7][W17-2]Reuben Tan

### PR DESCRIPTION
**Pull request by RTJL**

### CODE REVIEW

**Comment by etjk** [comment]
Excellent job @RTJL 
Maybe you could include more test case such as adding special characters.

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -79,14 +79,14 @@ Format: `find KEYWORD [MORE_KEYWORDS]`
 
 [NOTE]
 ====
-The search is case sensitive, the order of the keywords does not matter, only the name is searched,
+The search is not case sensitive, the order of the keywords does not matter, only the name is searched,
```
Comment (amrut-prabhu) [inline docs/UserGuide.adoc:82]: Good job for updating the User Guide.

File: src/seedu/addressbook/commands/FindCommand.java [hunk 2]
```diff
@@ -50,11 +51,22 @@ public CommandResult execute() {
...
         return matchedPersons;
     }
 
+    /**
+     * Converts a set of strings to be all lower cased.
+     * @param setOfStrings Set containing Strings to be converted to lower cased.
+     * @return Set of lower cased Strings.
+     */
+    private Set<String> setToLowerCase(Set<String> setOfStrings) {
+        setOfStrings = setOfStrings.stream().map(String::toLowerCase).collect(Collectors.toSet());
```
Comment (amrut-prabhu) [inline src/seedu/addressbook/commands/FindCommand.java:68]: Good use of Java streams!

Comment (amrut-prabhu) [inline src/seedu/addressbook/commands/FindCommand.java:68]: Can possibly name this `lowerCaseSetOfStrings` (though that may be too long) or something similar instead

File: test/expected.txt [hunk 3]
```diff
@@ -213,8 +213,9 @@
 || 0 persons listed!
 || ===================================================
 || Enter command: || [Command entered:  find betsy]
+|| 	1. Betsy Choo Tags: [secretive]
```
Comment (amrut-prabhu) [inline test/expected.txt:216]: Good work for updating the I/O tests

File: test/java/seedu/addressbook/commands/FindCommandTest.java [hunk 4]
```diff
@@ -2,11 +2,7 @@
 
 import static org.junit.Assert.assertEquals;
 
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```
Comment (amrut-prabhu) [inline test/java/seedu/addressbook/commands/FindCommandTest.java:5]: Avoid bulk imports.

Tip: You can change your IntelliJ settings to automate this for you

File: test/java/seedu/addressbook/commands/FindCommandTest.java [hunk 5]
```diff
@@ -22,11 +18,20 @@
 
     @Test
     public void execute() throws IllegalValueException {
-        //same word, same case: matched
+        //same word, all same case: matched
         assertFindCommandBehavior(new String[]{"Amy"}, Arrays.asList(td.amy));
 
-        //same word, different case: not matched
-        assertFindCommandBehavior(new String[]{"aMy"}, Collections.emptyList());
+        //same word, all upper case: matched
```
Comment (amrut-prabhu) [inline test/java/seedu/addressbook/commands/FindCommandTest.java:24]: Good job for adding additional JUnit test cases!

File: test/java/seedu/addressbook/commands/FindCommandTest.java [hunk 6]
```diff
@@ -22,11 +18,20 @@
 
     @Test
     public void execute() throws IllegalValueException {
-        //same word, same case: matched
+        //same word, all same case: matched
```
Comment (amrut-prabhu) [inline test/java/seedu/addressbook/commands/FindCommandTest.java:21]: Add a space after `//` in comments

**Review by amrut-prabhu — APPROVED** [review]
Good work on the LO @RTJL 
Please close the PR after reading the comments.

**Pull request closed without merging**